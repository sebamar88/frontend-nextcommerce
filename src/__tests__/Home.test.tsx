import Home from "../pages/index";
import { render } from "@testing-library/react";

describe("Home", () => {
    it("renders correctly", () => {
        const { container } = render(<Home />);
        expect(container).toMatchSnapshot();
    });
});
