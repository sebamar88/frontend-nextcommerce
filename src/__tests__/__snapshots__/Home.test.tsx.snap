// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div>
  <main
    class="max-w-3/4 mx-auto"
  >
    <h1
      class="text-4xl font-bold"
    >
      Hello World
    </h1>
  </main>
</div>
`;
